advocacy_docs/supported-open-source/patroni/installing_on_rhel.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: 'Installing Patroni on RedHat compatible systems'
+navTitle: 'Installing Patroni on RHEL'
```

Comment:
This page isn't about installing Patroni, but is about installing the database server instead.

Reply:
I'll add the patroni instructions. They seem to have been missed in a copy and paste

Reply:
Fixed in commit 4d2dd18776747591ca560e81e0b33a3da4c462eb